feature-libs/quote/assets/translations/en/quote.i18n.ts

```diff
@@ -107,6 +107,20 @@ export const quote = {
           successMessage: 'Quote cancelled',
         },
       },
+      expired: {
+        edit: {
+          title: 'Confirm Edit Quote {{ code }}?',
+          confirmNote: 'Are you sure you want to edit this expired quote?',
+          warningNote:
+            'This Quote is Expired. Editing this Quote will prevent Checkout until new edits are approved.',
+        },
+        requote: {
+          title: 'Recreate Quote Request {{ code }}?',
+          confirmNote: 'Are you sure you want to recreate this quote request?',
+          warningNote:
+            'This Quote is Expired. Re-Quoting it will create a new quote request with same contents, which can be edited and submitted for approval afterwards.',
```

Comment:
```suggestion
            'This Quote is expired. Re-quoting it will create a new quote request with same contents, which can be edited and submitted for approval afterwards.',
```